features/CLI/CommandLine.java

```diff
@@ -0,0 +1,143 @@
+import java.io.IOException;
+import java.util.Scanner;
+
+import org.json.simple.parser.ParseException;
+
+public class CommandLine extends HelperMethods {
+	
+	CommandLine(){
+	}
+
+	public static void commandInterface() throws IOException, ParseException {
```

Comment:
I can't seem to find a main function, is this going to be the entry point that main calls then? And should main catch the IOException and PaeseException?

Reply:
The commandInterface method will be called from the main in develop, yes.